### PR TITLE
Swift 5.1

### PR DESCRIPTION
**Pull request by stephencelis**
🎉

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM swift:5.0.2
+FROM swift:5.1
```
Comment (mbrandonw) [inline Dockerfile]: is it possible to use 5.1.3 for this too? I thought a bunch of foundation stuff was fixed on linux in releases after 5.1

File: Sources/Logger/Logger.swift [hunk 2]
```diff
@@ -1,106 +0,0 @@
...
-    _ message: @autoclosure () -> A,
-    file: StaticString = #file,
-    line: UInt = #line) {
-
-    let file = String(String(describing: file).split(separator: "/").last!)
-
-    if level.rawValue >= self.level.rawValue {
-      switch level {
-      case .debug, .info, .warn:
-        print("[\(level)] \(file):\(line): \(message())", to: &self.output)
```
Comment (mbrandonw) [inline Sources/Logger/Logger.swift]: we're losing the `file` and `line` stuff in the new logger. I think we should get that back cause it's been useful in the past.

File: Dockerfile (hunk 1 above)
Comment (stephencelis) [reply Dockerfile]: Per this page I believe 5.1 is the same as 5.1.3.

Comment (stephencelis) [reply Dockerfile]: https://hub.docker.com/_/swift

File: Sources/Logger/Logger.swift (hunk 2 above)
Comment (stephencelis) [reply Sources/Logger/Logger.swift]: done!

**Pull request merged**